src/PowerShellEditorServices/Workspace/ScriptFile.cs

```diff
@@ -50,7 +51,18 @@ public string Id
         /// <summary>
         /// Gets the path which the editor client uses to identify this file.
         /// </summary>
-        public string ClientFilePath { get; private set; }
+        public string ClientFilePath { get; private set; } 
+
+        /// <summary>
+        /// Gets the ClientPath in LSP DocumentUri form.  The ClientPath property must not be null.
+        /// </summary>
+        public string ClientPathAsDocumentUri
+        {
+            get 
+            {
+                return Workspace.ConvertPathToDocumentUri(this.ClientFilePath);
```

Comment:
Could we make it return `null` or `string.Empty` if `ClientFilePath` is null?  `ScriptFile` is actually accessible in a few ways without reflection.

```suggestion
                return this.ClientFilePath == null
                    ? string.Empty
                    : Workspace.ConvertPathToDocumentUri(this.ClientFilePath);
```

Reply:
Done.